projects/plugins/jetpack/class.jetpack.php

```diff
@@ -4069,15 +4085,15 @@ function admin_menu_order() {
 	 *
 	 * @return array
 	 */
-	function jetpack_menu_order( $menu_order ) {
+	public function jetpack_menu_order( $menu_order ) {
 		$jp_menu_order = array();
 
 		foreach ( $menu_order as $index => $item ) {
-			if ( $item != 'jetpack' ) {
+			if ( 'jetpack' !== $item ) {
 				$jp_menu_order[] = $item;
 			}
 
-			if ( $index == 0 ) {
+			if ( 0 == $index ) { // phpcs:ignore WordPress.PHP.StrictComparisons.LooseComparison
```

Comment:
Unless we're worried about weird edge cases that were fixed in PHP 7.2, it should be safe to make this strict.

The weird edge cases are things like assigning a numeric property to an object then casting the object to an array, e.g.
```php
$zero = 0;
$obj->$zero = "zero";
$arr = (array) $obj;
```

Although... is this even used? I can't find any reference to it, unless the function name is being concatenated oddly somehow.

Reply:
Looks like this hasn't been used since bdf1614aa38d6a8c2026e3baa04872d94a032399 . I'll follow-up in another PR to remove it and some related functions that aren't used anymore.

Reply:
https://github.com/Automattic/jetpack/pull/22310